scripts/superdesk-archive/module.js

```diff
@@ -140,7 +140,24 @@ import BaseListController from './controllers/baseList';
          * @param {Object} item
          */
         this.unspike = function(item) {
```

Comment:
This is perfect as is, but just for fun, it could be:
```js
this.unspike = item => getUnspikeDestination().then(config => unspike(item, config));
```